bin/catalog-api-order-minion

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+lib = File.expand_path("../lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+STDOUT.sync = true
+
+require "bundler/setup"
+require "catalog/minion"
+
+def parse_args
+  require 'optimist'
+  opts = Optimist.options do
+    opt :queue_host, "Hostname of the Platform's kafka queue", :type => :string,
```

Comment:
More of just a general question, how are the Optimist options used? Is it GNU style (`--queue_host`) or just `QUEUE_HOST` as an env var?

Reply:
@lindgrenj6 They're environment variables. You can see them set up here: https://console.insights-dev.openshift.com/console/project/db-catalog/browse/dc/catalog-api-minion?tab=environment